README.md

```diff
@@ -2598,22 +2577,24 @@ namespace `Service/`, but in separate namespaces such as `Processor/`.
 ```php
 <?php
 
+declare(strict_types=1);
+
 // namespace Acme\FooBundle\Service; // Incorrect
 namespace Acme\FooBundle\Processor; // Correct
 
 use Doctrine\ORM\EntityManagerInterface;
 
 class FooProcessor
 {
-    protected $entityManager;
-    protected $remoteServiceClient;
+    protected EntityManagerInterface $entityManager;
+    protected RemoteServiceClient $remoteServiceClient;
```

Comment:
Would be good to review all examples that have items with "protected" visibility. "Protected" visibility is only needed in case of inheritance and our conventions state that we should use composition instead, therefore in most cases such examples are invalid